seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumRun.scala

```diff
@@ -171,18 +163,31 @@ private[selenium] object SeleniumRun {
   private def maybeCleanupDriver(d: WebDriver, config: SeleniumJSEnv.Config) =
     if (!config.keepAlive) d.close()
 
-  private def htmlPage(scripts: Seq[java.net.URL]): String = {
-    val scriptTags =
-      scripts.map(path => s"<script src='${path.toString}'></script>")
+  private def htmlPage(setupJsUrl: URL, input: Seq[Input], materializer: FileMaterializer): String = {
+    val setupJs = s"<script src='${setupJsUrl.toString}'></script>"
+
+    val tags = input.map {
+      case Input.Script(path) => makeTag(path, "text/javascript", materializer)
+      case Input.ESModule(path) => makeTag(path, "module", materializer)
+      case _ => throw new UnsupportedInputException(input)
```

Comment:
Align the arrows? (see https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#pattern-matching)